packages/@contentful--app-scripts/lib/upload/create-app-bundle.js

```diff
@@ -9,8 +9,8 @@ async function createAppBundleFromUpload(settings, appUploadId) {
   console.log(settings.definition);
   const clientSpinner = ora('Verifing your upload').start();
   const client = createClient({ accessToken: settings.accessToken });
-  const org = await client.getOrganization(settings.organisation.value);
-  const appDefinition = await org.getAppDefinition(settings.definition.value);
+  const organization = await client.getOrganization(settings.organisation.value);
```

Comment:
Manuel was talking about `organisation` with an `s` here.

Reply:
The other way round. We use American English in contentful. So: `organization`.
Sorry for being unspecific.

Reply:
Ahhh I see, okay I changed it now everywhere to `organization`